From: Andrew Morton <akpm@linux-foundation.org>

alpha:

include/linux/time.h:97: error: expected '=', ',', ';', 'asm' or '__attribute__' before '__read_mostly'

Cc: Thomas Gleixner <tglx@linutronix.de>
Cc: Chris Wright <chrisw@sous-sol.org>
Cc: Ingo Molnar <mingo@elte.hu>
Cc: john stultz <johnstul@us.ibm.com>
Cc: David Miller <davem@davemloft.net>
Cc: Roman Zippel <zippel@linux-m68k.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 include/linux/time.h |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.22/include/linux/time.h
===================================================================
--- linux-2.6.22.orig/include/linux/time.h	2007-07-24 08:56:53.000000000 +0200
+++ linux-2.6.22/include/linux/time.h	2007-07-24 08:56:53.000000000 +0200
@@ -2,6 +2,7 @@
 #define _LINUX_TIME_H
 
 #include <linux/types.h>
+#include <linux/cache.h>
 
 #ifdef __KERNEL__
 # include <linux/seqlock.h>
